br/pkg/metautil/metafile.go

```diff
@@ -55,10 +55,12 @@ const (
 	MetaV2
 )
 
+// CreateMetaFileName creates a meta file name.
```

Comment:
```suggestion
// CreateMetaFileName is the name of meta file.
```

The function name is not so good...